SeaBattleTest/Ships.cs

```diff
@@ -0,0 +1,115 @@
+﻿using SeaBattle;
+using System;
+using System.Collections.Generic;
+using SeaBattle;
+using FluentAssertions;
+using System.Drawing;
+using System.Data.Common;
+using System.Runtime.CompilerServices;
+
+namespace SeaBattleTest
+{
+
+    [TestClass]
+    public class Ships
+    {
+        [TestMethod]
+        public void MoveTo_DontMoves_ifPlaceAlreadyOccupied()
+        {
+            //arrange
+            Sea sea = new Sea(10, 10);
+            List<Point> MoveToCoords = new List<Point>(new Point[] { new Point(5, 5), new Point(5, 6), new Point(5, 7), new Point(5, 8) });
+
+            List<Point> HybrydCoords = new List<Point>(new Point[] { new Point(3, 3), new Point(3, 4), new Point(3, 5), new Point(3, 6) });
+            HybridShip hybrydo = new HybridShip("Kolipso", HybrydCoords);
+            List<Point> WarCoords = new List<Point>(new Point[] { new Point(5, 5), new Point(5, 6), new Point(5, 7), new Point(5, 8)});
+            BattleShip warrior = new BattleShip("Warr", WarCoords);
+            //act
+            sea.AddShip(hybrydo);
+            sea.AddShip(warrior);
+
+            //assert
+            hybrydo.Invoking(hybrydo => hybrydo.MoveTo(MoveToCoords, sea)).Should().Throw<ArgumentOutOfRangeException>();
+        }
+
+        [TestMethod]
+        public void MoveTo_DontMoves_ifSpeedLowerThanDistance()
+        {
+            //arrange
+            Sea sea = new Sea(10, 10);
+            
+            List<Point> WarCoords = new List<Point>(new Point[] { new Point(1, 1), new Point(1, 2)});
+            BattleShip warrior = new BattleShip("Koalisto", WarCoords);          
+            warrior.Speed = 5;
+            sea.AddShip(warrior);
+
+            List<Point> MoveToCoords = new List<Point>(new Point[] { new Point(8, 9), new Point(9, 9) });
+            //act
+
+            //assert
+            warrior.Invoking(warrior => warrior.MoveTo(MoveToCoords, sea)).Should().Throw<Exception>("Speed can be lower than TraveDistance");
```

Comment:
Why do you put an error message as an argument? If you want to test if the Exception has this message in it, you have to use .WithMessage() method, .Throw() method takes the "because" argument which is intended to describe, why this assertion is required. So your message doesn't make sense.

Reply:
Yes, I know, I have some exceptions with "WithMessage()", it was ArgumentOutOfRangeException(with no text obv), but why it changed I have no idea, changed.